apps/api/e2e/tests/user.test.ts

```diff
@@ -525,7 +525,6 @@ describe("User API", () => {
     expect(defaultAgents[0]?.stats?.completedCompetitions).toBe(0);
     expect(defaultAgents[0]?.stats?.totalTrades).toBe(0);
     expect(defaultAgents[0]?.stats?.totalVotes).toBe(0);
-    // TODO: once `agent_rank` + `competitions_leaderboard` are implemented, these should be defined
```

Comment:
The default values are undefined (not sure the best representation but it's what we have so far)